external-api/src/interfaces/number.interface.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * Defines the interfaces for the response of the number generator.
+ *
+ * @interface NumberGeneratorResponse
+ * @property {number} status - The status code of the response.
+ * @property {Array} data - An array of objects containing the generated numbers.
+ * @property {number} data.id - The ID of the generated number.
+ * @property {number} data.maskedId - The masked ID of the generated number.
+ * @property {number} data.type - The type of the generated number.
+ * @property {string} data.createdBy - The creator of the generated number.
+ * @property {string} data.createdDatetime - The creation datetime of the generated number.
+ * @property {string} data.requestingSystem - The system that requested the generation of the number.
+ */
+export interface NumberGeneratorResponse {
+  readonly status: number;
+  readonly data: Array<{
+    id: number;
+    maskedId: number;
```

Comment:
out of curiosity what does this represent?

Reply:
This is the actual number as string